xdsl/dialects/ltl.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+from typing import Annotated
+
+from xdsl.dialects.builtin import IntegerType, Signedness
+from xdsl.ir import Attribute, Dialect, ParametrizedAttribute, SSAValue, TypeAttribute
+from xdsl.irdl import (
+    AnyOf,
+    ConstraintVar,
+    IRDLOperation,
+    irdl_attr_definition,
+    irdl_op_definition,
+    result_def,
+    var_operand_def,
+)
+
+"""
+Implementation of the LTL dialect by CIRCT. Documentation: https://circt.llvm.org/docs/Dialects/LTL/
+"""
```

Comment:
```suggestion
"""
Implementation of the LTL dialect by CIRCT.

https://circt.llvm.org/docs/Dialects/LTL/
"""
```

Reply:
Moreover, module docstrings should be at the very top of the file.
Please move it there.